tests/scenarios/k8s/base/mpi-job.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: kubeflow.org/v2beta1
+kind: MPIJob
+metadata:
+  name: nccl-tests
+  labels:
+    app: nccl-tests
+spec:
+  slotsPerWorker: 8
+  runPolicy:
+    cleanPodPolicy: Running
+    activeDeadlineSeconds: 666
+  mpiReplicaSpecs:
+    Launcher:
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: nccl-tests
+        spec:
+          restartPolicy: OnFailure
+          containers:
+          - image: ghcr.io/Azure/aks-rdma-infiniband/nccl-tests
+            name: nccl
+            env:
+            - name: OMPI_ALLOW_RUN_AS_ROOT
+              value: "1"
+            - name: OMPI_ALLOW_RUN_AS_ROOT_CONFIRM
+              value: "1"
+            command: ["/bin/bash", "-c"]
+            args:
+            - |
+              set -xe
+              until mpirun -np 16 -x LD_LIBRARY_PATH -bind-to none /usr/bin/nvidia-smi; do sleep 5; done
+              mpirun \
+                -np 16 \
+                -bind-to none \
+                -x NCCL_DEBUG=INFO \
+                -x NCCL_NET_GDR_LEVEL=PHB \
```

Comment:
I am wondering if this environment variable will have impact to the performance. In the [docs](https://docs.nvidia.com/deeplearning/nccl/user-guide/docs/env.html#nccl-net-gdr-level-formerly-nccl-ib-gdr-level), it mentions:

> PHB : Use GPU Direct RDMA when GPU and NIC are on the same NUMA node. Traffic will go through the CPU.

Have you tried different values?

Reply:
We will go with the value `SYS` for now.